envctl/env_scripts/java/kubernetes.sh

```diff
@@ -58,7 +58,7 @@ attach_or_create_gke_cluster(){
     echo "cluster not found. creating..."
     gcloud container clusters create $SERVICE_NAME \
       --zone $ZONE \
-      --scopes=gke-default,pubsub \
+      --scopes=gke-default,pubsub,cloud-platform \
```

Comment:
This is interesting: if we add cloud-platform scope, I think we don't need anything else as it already includes everything in it. 
Should we try to narrow it down to minimum scopes to make this work? Or, alternatively, if too many are required, reduce it to cloud-platform only.

Reply:
good point, I changed it to just `cloud-platform` for now